src/components/Icon/NftAsset.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Svg, {Path} from 'react-native-svg'
+
+type Props = {
+  size?: number
+  color?: string
+}
+
+export const NftAsset = ({size = 36, color = 'black'}: Props) => (
```

Comment:
Would you be so kind to check with the design team, it looks a little bit different from the TabBar that is on Figma.